src/validator/profile.ts

```diff
@@ -1,5 +1,6 @@
 import { Request, Response, NextFunction } from 'express';
 export const updateProfileValidator = (req: Request, res: Response, next: NextFunction) => {
+  // TODO is only checking for length but not what is allowed etc.
```

Comment:
I didnt know that regex cant migtate sql injections but if that is the case one possibility would be to escape the description and store it in the database and when we return it we unescape it again. 
Maybe we can still restrict some characters to avoid stored xss.